### PR TITLE
fix(PX-4402): use small buttons

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
The type of this PR is: Bugfix

This PR resolves [PX-4402]

### Description

This PR fixes text splitting for action buttons on the order history page

### PR Checklist (tick all before merging)

- [X] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [X] I have tested my changes on **iOS** and **Android**.
- [X] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [X] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [X] I have documented any follow-up work that this PR will require, or it does not require any.
- [X] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [X] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fixes text splitting for action buttons - kirill-zubarau

</details>

Before
image

Now
image

[PX-4402]: https://artsyproduct.atlassian.net/browse/PX-4402

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fixes text splitting for action buttons - kirill-zubarau)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against eb857f19662fc110b679409ad870dcd656523512
</p>

File: src/lib/Scenes/OrderHistory/OrderHistoryRow.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ export const OrderHistoryRow: React.FC<OrderHistoryRowProps> = ({ order }) => {
           <Box width="50%" paddingRight={0.5}>
```
Comment (ansor4) [inline src/lib/Scenes/OrderHistory/OrderHistoryRow.tsx:67]: After consulting with Gideon, we landed on two full width buttons! Thanks for bearing with the back and forth!

https://artsy.slack.com/archives/C9ZJYFDNF/p1632406953123800

**Review by ansor4 — CHANGES_REQUESTED** [review]
Update for different direction in design 🙏 🙏

**Review by ansor4 — APPROVED** [review]
🎉

**Comment by ansor4** [comment]
@kiryl-zubarau looks like there's a merge conflict, otherwise good to go!

**Review by MounirDhahri — APPROVED** [review]
⛴

**Pull request merged**